File: control-flow/Q116/main.go
package main

import "fmt"

func main() {
	var n int
	var sum int
	fmt.Print("Enter n: ")
	fmt.Scan(&n)
	for i := 1; i <= n; i++ {
		if i < n {
			fmt.Print(i, " + ")
		} else {
			fmt.Print(i, " = ")
		}
		sum += i
	}
	fmt.Print(sum)
}

File: control-flow/Q116/main.rs
use std::io;

fn main() {
    let mut input = String::new();

    println!("Enter n:");
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let n: i32 = input.trim().parse().expect("Not a valid number");

    let mut sum: i32 = 0;

    for i in 1..=n {
        if i < n {
            print!("{} + ", i);
        } else {
            print!("{} = ", i);
        }
        sum += i;
    }

    print!("{}", sum);
}